<launch>

    <arg name="path" default="/home/dearmoon/datasets/NWU/"/>
    <arg name="scene" default="日雪不颠簸高速/"/>
    <arg name="group" default="222/"/>

    <node name="CustomMsgFillsing" type="CustomMsgFilling" pkg="rosbag_tools" output="screen">
        <param name="outputPath" value="$(arg path)$(arg scene)enhancing_v2/$(arg group)FilledCustomMsg.bag"/>
    </node>

    <node name="player_1" pkg="rosbag" type="play"
        args="--clock
        $(arg path)$(arg scene)enhancing/radar_lidar_input.bag
        --topic /livox/lidar">
    </node>

    <node name="player_2" pkg="rosbag" type="play"
        args="--clock
        $(arg path)$(arg scene)enhancing_v2/$(arg group)/CustomMsg.bag
        --topic /livox/lidar">
        <remap from="livox/lidar" to="/RegionGrowing/CustomMsg"/>
    </node>

</launch>